### PR TITLE
fixes #165: fixed issue with SSID containing spaces in imagebuilder.sh

### LINKED ISSUE #165: imagebuilder.sh cannot handle SSID with spaces
Using the `imagebuilder.sh` with an SSID which contains spaces does not work.

    sudo ./dustcloud/devices/xiaomi.vacuum/firmwarebuilder/imagebuilder.sh -f v11_001518.pkg -k id_rsa.pub --unprovisioned wpa2psk --ssid 'FRITZ!Box Fon WLAN 7590' --psk password

### PR DESCRIPTION
**Pull request by jansohn**
see #165

### CODE REVIEW

File: devices/xiaomi.vacuum/firmwarebuilder/imagebuilder.sh [hunk 1]
```diff
@@ -291,8 +291,8 @@ if [ "$UNPROVISIONED" = true ]; then
         chmod +x ./etc/rc.local
         cp $BASEDIR/unprovisioned/wpa_supplicant.conf.wpa2psk ./opt/unprovisioned/wpa_supplicant.conf
 
-        sed -i 's/#SSID#/'$SSID'/g' ./opt/unprovisioned/wpa_supplicant.conf
-        sed -i 's/#PSK#/'$PSK'/g' ./opt/unprovisioned/wpa_supplicant.conf
+        sed -i 's/#SSID#/'"$SSID"'/g' ./opt/unprovisioned/wpa_supplicant.conf
+        sed -i 's/#PSK#/'"$PSK"'/g' ./opt/unprovisioned/wpa_supplicant.conf
```
Comment (cryptomilk) [inline devices/xiaomi.vacuum/firmwarebuilder/imagebuilder.sh]: This hunk is wrong and not needed. Did you actually look in the final file?

Comment (jansohn) [reply devices/xiaomi.vacuum/firmwarebuilder/imagebuilder.sh]: Yes, it was needed otherwise I got an SED error message...

Comment (cryptomilk) [reply devices/xiaomi.vacuum/firmwarebuilder/imagebuilder.sh]: You're right, bash is often very confusing :-)

**Comment by dgiese** [comment]
Good job, thank you ;)

**Pull request merged**